app/resources/api/v3/public/card_resource.rb

```diff
@@ -8,7 +8,9 @@ class Api::V3::Public::CardResource < JSONAPI::Resource
         attributes :advancement_requirement, :agenda_points, :base_link, :cost
         attributes :deck_limit, :influence_cost, :influence_limit, :memory_cost
         attributes :minimum_deck_size, :strength, :stripped_text, :text, :trash_cost
-        attributes :is_unique, :display_subtypes, :updated_at
+        attributes :is_unique, :display_subtypes
+        attribute :latest_printing_id
```

Comment:
I have been lightly trying to keep "real" properties separate from "calculated" properties.

Would you keep the original line and put your new property on it's own line?